internal/querycoordv2/job/job_load.go

```
@@ -205,6 +196,15 @@ func (job *LoadCollectionJob) Execute() error {
 		return errors.Wrap(err, msg)
 	}
 
+	// 5. update next target, no need to rollback if pull target failed, target observer will pull target in periodically
+	_, err = job.targetObserver.UpdateNextTarget(req.GetCollectionID())
+	if err != nil {
+		msg := "failed to update next target"
+		log.Error(msg, zap.Error(err))
```

Ignore this error and downgrade the log level to WARN